client-app/shared/layout/components/language-selector/language-selector.vue

```diff
@@ -98,4 +104,16 @@ function select(locale: string) {
   saveLocaleAndReload(locale);
   hideList();
 }
+
+function getFlagEmoji(countryCode: string): string {
+  // commonly accepted approach to convert country code to flag emoji
+  return countryCode.toUpperCase().replace(/./g, (char) => String.fromCodePoint(char.charCodeAt(0) + 127397));
+}
 </script>
+
+<style scoped>
+.country-flag {
```

Comment:
Specify font-flag class in tailwing.config.ts and use it in template

https://tailwindcss.com/docs/font-family#customizing-your-theme